R/provider-ollama.R

```diff
@@ -49,19 +49,19 @@ chat_ollama <- function(system_prompt = NULL,
   )
 }
 
-ollama_models <- function() {
-  req <- request("http://localhost:11434/api/tags")
+ollama_models <- function(base_url = "http://localhost:11434") {
+  req <- request(file.path(base_url, "api/tags"))
```

Comment:
Would be slightly cleaner to use `req_url_path()` to add `api/tags` to the `base_url`

Reply:
No problem.